Python/pdf_to_mongodb/pdfmongo.py

```diff
@@ -0,0 +1,22 @@
+import tabula
+import pandas as pd
+from pymongo import MongoClient
+
+pdf_path = "data.pdf";
```

Comment:
take input of the PDF file instead of hardcoding it. You can use command-line arguments parser or directly using input() that's your choice.